### DISCUSSION TITLE
Values for Model Download cell?

### DISCUSSION THREAD

**Discussion by aleph1** [TheLastBen/fast-stable-diffusion#2135] [General]
I have tried various values for the Model Download cell, but keep getting an error “Wrong path, use the colab file explorer to copy the path”. Which values should I be using for the following when trying to utilize https://huggingface.co/runwayml/stable-diffusion-v1-5?

- Path_to_HuggingFace
- MODEL_PATH
- MODEL_LINK

It might be worth clarifying this in the instructions in the notebook as the descriptions for each value are a bit vague.

**Comment by TheLastBen** [comment]
if you want the default model, simply leave everything empty and run the cell

and I did clarify in the notebook how to use a huggingface link with an example

**Reply by coudys** [reply]
I tried to train using other than default runwayml 1.5 model but that notebook doesnt accept huggingface path as I would have expected. Is the model not compatible with training or the path needs to be somehow modified? https://huggingface.co/Getspastic/A-ZovyaPhotoreal
"Getspastic/A-ZovyaPhotoreal"

**Reply by TheLastBen** [reply]
post a screenshot of the cell output

**Reply by coudys** [reply]
![colab_lastben_fastdreambooth_model_error](https://github.com/TheLastBen/fast-stable-diffusion/assets/102684605/6d33eb0e-7750-4b85-ae3e-4284a43bcb76)

**Reply by TheLastBen** [reply]
there is no diffusers model in the link, use the safetensors link instead https://huggingface.co/Getspastic/A-ZovyaPhotoreal/resolve/main/aZovyaPhotoreal_v1.safetensors

put it in MODEL_LINK and tick the box "safetensors", don't forget to empty the other input boxes

**Reply by coudys** [reply]
that worked, great, thank you!
